components/constellation/constellation.rs

```diff
@@ -1405,10 +1405,6 @@ impl<Message, LTF, STF> Constellation<Message, LTF, STF>
 
             let window_size = old_pipeline.and_then(|old_pipeline| old_pipeline.size);
 
-            if let Some(old_pipeline) = old_pipeline {
```

Comment:
@jdm: thoughts? Should we suspend timers when a document is navigated from, or when it becomes inactive? (I think the latter, but you may have opinions.)